### PR TITLE
Initial implementation.

### PR DESCRIPTION
**Pull request by SiahaanBernard**
Initial implementation of installing mongodb client version 3.2.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+galaxy_info:
+  description: Installs and configures MongoDB shell and mongodb tools 4.0
```
Comment (rizkidoank) [inline meta/main.yml]: is it 4.0 or 3.2.12?

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        name: "{{ item }}"
+        state: present
+        update_cache: yes
+        cache_valid_time: "{{ apt_cache_valid_time | default(omit) }}"
+      with_items: "{{ mongodb_packages }}"
+
+  become: true
+  become_method: sudo
+  tags:
+    - mongodb-4.0
```
Comment (rizkidoank) [inline tasks/main.yml]: 4.0 or 3.2?

File: tests/requirements.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+- name: ansible-apt_sources_list
+  src: https://github.com/traveloka/ansible-apt_sources_list
+  scm: git
+  version: v1.1.0
+
```
Comment (rizkidoank) [inline tests/requirements.yml]: NL nitpick

**Comment by SiahaanBernard** [comment]
Testing output:
Target:  ssh://vagrant@127.0.0.1:2222

  System Package mongodb-org-shell
     ✔  should be installed
     ✔  version should cmp >= "3.2"
  System Package mongodb-org-tools
     ✔  should be installed
     ✔  version should cmp >= "3.2"

Test Summary: 4 successful, 0 failures, 0 skipped

**Pull request merged**